internal/lookout/repository/store_test.go

```diff
@@ -979,7 +980,7 @@ func getPriority(t *testing.T, db *goqu.Database, jobId string) float64 {
 
 func getJob(t *testing.T, db *goqu.Database, jobId string) *api.Job {
 	var job api.Job
-	jobJson := ParseNullString(selectNullString(t, db, fmt.Sprintf("SELECT job FROM job WHERE job_id = '%s'", jobId)))
+	jobJson := ParseNullString(selectNullString(t, db, fmt.Sprintf("SELECT orig_job_spec FROM job WHERE job_id = '%s'", jobId)))
```

Comment:
this should probably switch to proto marshall/unmarshall instead of jamming json in there